VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -71,17 +75,19 @@ public void onClick(View v) {
             }
         });
 
-        holder = createViewHolder(inflater, parent);
-
-        holder.typeView.setIcon(Iconify.IconValue.fa_comments_o);
-        holder.titleView.setText(R.string.discussion_title);
-        holder.subtitleView.setText(R.string.discussion_subtitle);
-        holder.rowView.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                environment.getRouter().showCourseDiscussionTopics(getActivity(), courseData);
-            }
-        });
+        if (courseData != null && !TextUtils.isEmpty(courseData.getCourse().getDiscussionUrl())) {
```

Comment:
Seems like this should also be checking the ``DISCUSSIONS_ENABLED`` config flag?